src/olympia/addons/api_urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import patterns, url
+
+from . import views
```

Comment:
I was undecided :)

We already have some "new" API things like signing in `views`, so I didn't want to decide just yet. My plan was to tackle this later.